tests/unit/client.py

```diff
@@ -637,36 +637,24 @@ def test_user_remove_error(self):
         with pytest.raises(FailedAPICallException):
             C.user.remove('sam03', 'test_proj03')
 
-    def test_user_set_privilege(self):
+    def test_user_set_admin(self):
         """Test changing a user's admin status """
         C.user.create('jimmy', '12345', 'regular')
         C.user.create('jimbo', '678910', 'admin')
         assert C.user.set_privilege('jimmy', 'admin') is None
```

Comment:
this should be `C.user.set_admin`
that should fix the client tests error you are having.